tests/plugins/test_rpmqa.py

```diff
@@ -53,14 +52,14 @@ def mock_logs(cid, **kwargs):
     {"ignore": True, "package_list": PACKAGE_LIST},
     {"ignore": False, "package_list": PACKAGE_LIST_WITH_AUTOGENERATED},
 ])
-def test_rpmqa_plugin(remove_container_error, ignore_autogenerated):
+def test_rpmqa_plugin(docker_tasker, remove_container_error, ignore_autogenerated):
     if MOCK:
         should_raise_error = {}
         if remove_container_error:
             should_raise_error['remove_container'] = None
         mock_docker(should_raise_error=should_raise_error)
 
-    tasker = DockerTasker()
+    tasker = docker_tasker
```

Comment:
There is no point in additional variable here

Reply:
ok